docs/unix/quickref.rst

```diff
@@ -3,6 +3,20 @@
 Quick reference for the UNIX and Windows ports
 ==============================================
 
+
+Installation
+------------
+
+MicroPython does not publish builds for MacOs, Unix, and Windows. If you need
+latest versions, you should build from source as outlined in the wiki page: 
+`Getting Started <https://github.com/micropython/micropython/wiki/Getting-Started>`_.
+Here some common ways to install unofficial builds:
+
+* MacOs: ``brew install micropython`` from https://formulae.brew.sh/formula/micropython
+* Unix: ``apt get micropython`` (or your favorite variant of apt, apt-get). MicroPython may already be in your distrbution.
```

Comment:
Not sure if the apt/apt-get bit is useful. `distrbution` -> `distribution`; but which distribution already has MicroPython?

Reply:
I don't know which distros have MicroPython, it was a comment from the forum. I'll remove.
I'll remove that parenthetical and assume `apt get` is clear enough.
Typo! Thanks for catching that!